sites/nau/src/backend/base/static/richie/images/components/checkmark.svg

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We don't need to replace the checkmark.svg. We can change its fill using normal CSS.
We need to fix this on Richie. I would send a PR.

Please update this PR and remove the checkmark.svg. This change is other change and shouldn't be on the same commit.